application/console/config/main.php

```diff
@@ -2,23 +2,8 @@
 return [
     'id' => 'app-console',
     'basePath' => dirname(__DIR__),
-    'bootstrap' => ['log', 'gii'],
-    'controllerNamespace' => 'console\controllers',
```

Comment:
Do we know if we plan to have any CLI commands for this? Let's see, I think the call to batch-sync with the ID store will be on a cron job, so we probably will. We may need this `controllerNamespace` property (unless you figured out it won't be necessary for some reason... which is quite possible).